src/Extension/ExemplifyExtension/README.md

```diff
@@ -28,7 +28,7 @@ This can then be modified to describe the behaviour for the method.
 
 Requires:
 
-* PhpSpec 2.0+
+* PhpSpec 3.0+
 * PHP 5.4+
 
 To use 'named constructor' examples , you need to use latest phpspec 2.1@dev.
```

Comment:
I think it's may be useful if an user wants to use Exemplify as standalone component